### DISCUSSION TITLE
Clicking source link in beatmap details should perform a google search not a beatmap search

### DISCUSSION THREAD

**Discussion by Sublimelime** [ppy/osu#20516] [Ideas] [answered]
Currently when you're looking at the details of a beatmap and click on the source (a link), you're sent to a beatmap search with the text of the source filled in. 

This could be useful for finding more beatmaps tagged with that same source, but it would perhaps be better to simply open a browser and google search the source text - at least personally when I click it I'm usually interested in **finding out more about the source itself** (being an anime or similar) and would expect it to give that info, not simply open a beatmap search.

Just an idea.

**Comment by peppy** [comment, marked as answer]
I don't think this is something we'll implement. Feels a bit weird. If someone wants to search they should be able to copy the source/tag to clipboard and perform the search manually.
